### PR TITLE
Add Trf 4.07b binary

### PR DESCRIPTION
**Pull request by remimarenco**

### CODE REVIEW

**Comment by remimarenco** [comment]
@bgruening If you can check that what I am doing is good that would be awesome! And then I will merge (or you can do it if you want to ;) )

File: recipes/trf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
+  url: https://tandem.bu.edu/trf/downloads/trf407b.macos64 # [osx]
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/trf/meta.yaml]: Build number can/should be 0 in a first iteration.

Comment (bgruening) [inline recipes/trf/meta.yaml]: license and a description is missing

File: recipes/trf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package:
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
+  url: https://tandem.bu.edu/trf/downloads/trf407b.macos64 # [osx]
```
Comment (bgruening) [inline recipes/trf/meta.yaml]: In general we recommend to compile from source, as you never know if these binaries are relocatable and will work on old Linux machines.
But if this is to complicated I'm fine with this, I have seen this before.

File: recipes/trf/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package:
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
```
Comment (bgruening) [inline recipes/trf/meta.yaml]: Btw. there is a [linux64] trick, just not sure how it is called. But linux is also linux-32

File: recipes/trf/meta.yaml (hunk 2 above)
Comment (remimarenco) [inline recipes/trf/meta.yaml]: Well...I wish I could take the sources! But did not find them :/ https://tandem.bu.edu/trf/downloads/

Comment (bgruening) [inline recipes/trf/meta.yaml]: Than it's ok ... maybe you can ask them for the source and enhance this recipe in build-1 if you get the source.

File: recipes/trf/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+build:
+  number: 0
+  skip: False
+
+test:
+  files:
+    - test-data/dbia3.fasta
+  commands:
+    - trf 2>&1 | grep Tandem > /dev/null
+    - trf dbia3.fasta 2 7 7 80 10 50 500 2>&1 | grep "Tandem" > /dev/null
```
Comment (bgruening) [inline recipes/trf/meta.yaml]: I like this test data example! Well done!

File: recipes/trf/meta.yaml (hunk 2 above)
Comment (remimarenco) [inline recipes/trf/meta.yaml]: Asked!

Comment (jxtx) [inline recipes/trf/meta.yaml]: Very unlikely to get the source. TRF is not under an open license. 

There are open alternatives – phobos, mreps, maybe VNTRseek. Would one of those work instead?

Comment (remimarenco) [inline recipes/trf/meta.yaml]: Will see with folks of my project if they are useful. Could be nice to add them anyway here! Thanks @jxtx

**Pull request merged**